src/pages/journal/journal-components/journal-slot/__tests__/journal-slot.spec.ts

```diff
@@ -0,0 +1,116 @@
+import { By } from '@angular/platform-browser';
+import { ConfigMock } from 'ionic-mocks-jest';
+
+describe('JournalSlotComponent', () => {
+    let fixture: ComponentFixture<JournalSlotComponent>;
+    let component: JournalSlotComponent;
+    const mockSlot = {
+          slotDetail: {
+            slotId: 1001,
+            start: '2018-12-10T09:07:00+00:00',
+            duration: 57,
+          },
+          vehicleSlotType: 'B57mins',
+          testCentre: {
+            centreId: 54321,
+            centreName: 'Example Test Centre',
+            costCode: 'EXTC1',
+          },
+          booking: {
+            candidate: {
+              candidateId: 101,
+              age: 17,
+              candidateName: {
+                title: 'Miss',
+                firstName: 'Florence',
+                lastName: 'Pearson',
+              },
+              driverNumber: 'PEARS015220A99HC',
+              gender: 'Female',
+              candidateAddress: {
+                addressLine1: '1 Station Street',
+                addressLine2: 'Someplace',
+                addressLine3: 'Sometown',
+                addressLine4: '',
+                addressLine5: '',
+                postcode: 'AB12 3CD',
+              },
+              primaryTelephone: '01234 567890',
+              secondaryTelephone: '04321 098765',
+              mobileTelephone: '07654 123456',
+            },
+            application: {
+              applicationId: 1234567,
+              bookingSequence: 3,
+              checkDigits: 1,
+              welshTest: false,
+              extendedTest: false,
+              meetingPlace: '',
+              progressiveAccess: false,
+              specialNeeds: 'Candidate has dyslexia',
+              entitlementCheck: false,
+              vehicleSeats: null,
+              vehicleHeight: null,
+              vehicleWidth: null,
+              vehicleLength: null,
+              testCategory: 'B',
+              vehicleGearbox: 'Manual',
+            },
+            previousCancellation: [
+              {
+                initiator: 'Act of nature',
+              },
+            ],
+          },
+        };
+        
+    beforeEach(async(() => {
+        TestBed.configureTestingModule({
+            declarations: [
+                JournalSlotComponent,
+                MockComponent(JournalTimeComponent),
+                MockComponent(JournalTestDetailsComponent),
+                MockComponent(JournalTestOutcomeComponent),
+                MockComponent(JournalCandidateComponent),
+            ],
+            imports: [IonicModule],
+            providers: [ { provide: Config, useFactory: () => ConfigMock.instance() } ]
+        }).compileComponents().then(() => {
+            fixture = TestBed.createComponent(JournalSlotComponent);
+            component = fixture.componentInstance;
+            component.slot = mockSlot;
+        })
+    }));
+
+    describe('Component Interaction', () => {
```

Comment:
Could you nest these under `describe('DOM', () => {` to make it consistent with other component tests please?

Reply:
Done.